### ISSUE TITLE
SideNav flash on load

### ISSUE THREAD

**Issue by francinelucca** [carbon-design-system/carbon-platform#898]
**Description**

Let's look into the sidenav flash when the page loads for the first time (theory: probably side nav data hasn't been set by the time the page is still loading, can we have the sidenav data initialize with default data or something?)

if this can be fixed, fix it 

**Steps to reproduce**

How do we reproduce the error? Please be as specific as you can.

1. Load page for the first time. Pay attention to the left where the sidenav loads, it will load as blank first and then populate the data

**Comment by mattrosno** [comment]
@francinelucca I think we had an approach where we set default side navs per router path so we don't have to wait on context to initialize prior to showing something? Hardcoding paths in constants isn't something that feels right - but that seemed to work to get the right side nav shown immediately without waiting for layout context to initialize.

**Comment by mattrosno** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Platform-Development-624c3fc8b7972e001130fef7/issues/carbon-design-system/carbon-platform/898?planning-poker) with ZenHub @alisonjoseph @andreancardona @francinelucca @jdharvey-ibm

**Comment by mattrosno** [comment]
Low priority for now.
